integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -16,14 +16,14 @@ resources:
             jobStatus: '.color as $c | {"notbuilt": "created", "blue": "passing", "red": "failing"}[$c // "updated"]'
             timestamp: .time
           relations:
-            jobs: '.jobs | map(if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1])'
+            jobs: '. as $body | $body.jobs | map($body.name + " " + .name  | sub(" "; "-"; "g")) '
```

Comment:
@yairsimantov20, this is so because if two jobs "JobA" and "JobB" both have `main` branches, then the identifiers become "JobA-main" and "JobB-main"